tests/dragonfly/cluster_test.py

```diff
@@ -1433,7 +1433,6 @@ async def test_migration_with_key_ttl(df_factory):
     assert await nodes[1].client.execute_command("stick k_sticky") == 0
 
 
-@pytest.mark.skip("test is flaky")
```

Comment:
Is this change intentional?
(I assume that changing the ports does not fix the test?)

Reply:
Thx